common/men_at_arms_types/wc_cultural_maa_types.txt

```diff
@@ -0,0 +1,1704 @@
+	
+	stack = 75
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = mogu
+}
+
+swarm = {
+	type = skirmishers
+	
+	damage = 20
+	toughness = 20
+	pursuit = 25
+	screen = 10
+
+	terrain_bonus = {
+		desert = { toughness = 10 pursuit = 15 }
+		desert_mountains = { toughness = 20 pursuit = 10 }
+	}
+	
+	counters = {
+		heavy_infantry = 1
+	}
+
+	buy_cost = { gold = cost_80_recruitment_cost }
+	low_maintenance_cost = { gold = cost_80_low_maint_cost }
+	high_maintenance_cost = { gold = cost_80_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = swarm
+}
+
+technomage = {
+	type = caster
+	
+	damage = 60
+	toughness = 30
+	pursuit = 15
+	screen = 15
+	
+	terrain_bonus = {
+	}
+
+	counters = {
+		pikemen = 1
+		massive_infantry = 1
+		heavy_infantry = 1
+		caster = 1
+	}
+
+	buy_cost = { gold = cost_110_recruitment_cost }
+	low_maintenance_cost = { gold = cost_110_low_maint_cost }
+	high_maintenance_cost = { gold = cost_110_high_maint_cost }
+	
+	stack = 75
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = caster
+}
+
+tidehunter = {
+	type = skirmishers
+	
+	damage = 22
+	toughness = 18
+	pursuit = 20
+	screen = 0
+	
+	terrain_bonus = {
+		wetlands = { damage = 25 toughness = 10 pursuit = 20 }
+	}
+	
+	counters = {
+		skirmishers = 1
+		casters = 1
+	}
+	
+	buy_cost = { gold = cost_80_recruitment_cost }
+	low_maintenance_cost = { gold = cost_80_low_maint_cost }
+	high_maintenance_cost = { gold = cost_80_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = tidehunter
+}
+
+thornguard = { 
+	type = heavy_infantry
+	
+	damage = 37
+	toughness = 30
+	pursuit = 0
+	screen = 20
+	
+	terrain_bonus = {
+		hills = { damage = 15 }
+		desert = { damage = 20 }
+		drylands = { damage = 20 pursuit = 10 }
+		desert_mountains = { toughness = 20 pursuit = 15 }
+	}
+	
+	counters = {
+		pikemen = 1
+	}
+	
+	buy_cost = { gold = cost_120_recruitment_cost }
+	low_maintenance_cost = { gold = cost_120_low_maint_cost }
+	high_maintenance_cost = { gold = cost_120_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = thornguard
+}
+
+ursa = { 
+	type = heavy_infantry
+	
+	damage = 35
+	toughness = 40
+	pursuit = 5
+	screen = 20
+	
+	terrain_bonus = {
+		forest = { damage = 20 toughness = 10 pursuit = 20 }
+		hills = { damage = 15 pursuit = 15 }
+	}
+	
+	counters = {
+		skirmishers = 1
+	}
+	
+	buy_cost = { gold = cost_125_recruitment_cost }
+	low_maintenance_cost = { gold = cost_125_low_maint_cost }
+	high_maintenance_cost = { gold = cost_125_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = ursa
+}
+
+war_dinosaur = {
+	type = heavy_cavalry
+	
+	damage = 130
+	toughness = 60
+	pursuit = 20
+	screen = 10
+	
+	terrain_bonus = {
+		plains = { damage = 20 }
+		drylands = { damage = 5 }
+		hills = { damage = -10 }
+		jungle = { damage = 5 }
```

Comment:
Heavy Cavalry in vanilla get heavy debuffs if there is any terrain that is not flat.  I gave most large grounded units debuffs in the jungle because their size would only hurt them in that terrain.  Dinosaurs are massive, but they are native so I tried to balance with a modest +5.  I think I should just stay consistent with larger beings having debuffs in heavily inconsistent terrain.

Reply:
Ok, sounds fair to me.